labs/coding-102-rest-python-ga/3.md

```diff
@@ -77,20 +77,24 @@ To run this code sample:
     `cd \DevNetCode\~your-name~\coding-skills-sample-code\coding102-REST-python-ga`
```

Comment:
We can change over to the other repo name, dnav3-code.

Reply:
Will fix. Thanks.

Reply:
Nope, reversing based on talking to Hank. 😄